### DISCUSSION TITLE
Do image serve by ipx get cached in public folder?

### DISCUSSION THREAD

**Discussion by Nemure231** [nuxt/image#809] [Q&A]
I've heared enough discussion in internet, that saying if i placed image in assets folder i can get many benefits like build by buildtools (vite/webpack etc) and get cache and so on. Because of that, i start using https://github.com/ElMassimo/vite-plugin-image-presets for reducing my image size, and place them into the assets folder.

But today, i found this nuxt official module about image. I am curious because the documentation told me to place my image in public folder rather than assets.

My question is the benefits about caching image still same if i using nuxt-image module, use default provider (ipx), and place image in public folder? Or maybe i can get more benefits if i am use nuxt-image?

**Comment by procrates** [comment]
IPX lets you define sizes and format to optimize the image. You can also specify cache in nuxt config for the image folder in public.
